web/src/OverviewItemView.tsx

```diff
@@ -244,6 +254,24 @@ export function triggerUpdate(name: string, action: string) {
   })
 }
 
+export function toggleTriggerMode(name: string, mode: TriggerMode) {
+  incr("ui.web.toggleTriggerMode", { to_mode: mode.toString() })
+
+  let url = `//${window.location.host}/api/override/trigger_mode`
```

Comment:
fwiw, i don't think the window.location.host is necessary here, you can use a '/api/overide' url